src/pages/ContributorFields.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { TextField, Typography } from '@material-ui/core';
+import { Contributor, Resource } from '../types/resource.types';
+import { ErrorMessage, Field, FieldArray, FieldArrayRenderProps, FieldProps, FormikProps, FormikValues } from 'formik';
+import Button from '@material-ui/core/Button';
+import { createContributor, putContributorFeature } from '../api/resourceApi';
+import Paper from '@material-ui/core/Paper';
+import styled from 'styled-components';
+import DeleteIcon from '@material-ui/icons/Delete';
+import AddIcon from '@material-ui/icons/Add';
+
+const StyledPaper = styled(Paper)`
+  width: 100%;
+  padding: 3rem;
+`;
+
+const StyledField = styled(Field)`
+  display: inline-block;
+  margin-left: 2rem;
+  margin-right: 2rem;
+  padding: 1rem;
+`;
+const StyledDiv = styled.div`
+  display: inline-block;
+  margin-top: 1rem;
+  margin-bottom: 0.3rem;
+`;
+
+const StyledTextField = styled(TextField)`
+  width: 22rem;
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
+`;
+
+const StyledButton = styled(Button)`
+  position: relative;
+  vertical-align: bottom;
+`;
+
+interface ContributorFieldsProps {
+  resource: Resource;
+  formikProps: FormikProps<FormikValues>;
+  setAllChangesSaved: any;
+}
+
+const ContributorFields: FC<ContributorFieldsProps> = ({ resource, formikProps, setAllChangesSaved }) => {
+  const { t } = useTranslation();
+
+  const addContributor = (arrayHelpers: FieldArrayRenderProps) => {
```

Comment:
Hvis dere bare har en type i lista kan man angi type her også, så vil man få feil om man sender inn noe feil i `push()` feks. men det kommer naturligvis an på hva som skal være i lista
```suggestion
  const addContributor = (arrayHelpers: FieldArrayRenderProps<Contributor>) => {
```

Reply:
Får feilmedling "TS2315: Type 'FieldArrayRenderProps' is not generic." når jeg prøver å legge det inn.

Reply:
stemmer, jeg som er på bærtur :)